R/lasnormalize.r

```diff
@@ -77,7 +77,7 @@ lasnormalize = function(.las, dtm, keep_Zdtm=FALSE, ...)
   {
     normalized = LAS(data.table::copy(.las@data), .las@header)
 
-    normalized@data$Zdtm=raster::extract(dtm, normalized@data[,.(X,Y)], ...)
+    normalized@data[, Zdtm := raster::extract(dtm, normalized@data[,.(X,Y)], ...)][]
```

Comment:
That's better :-) . But it is exatly what `lasclassify` does. You should avoid duplicated code.